CONTRIBUTING.md

```diff
@@ -53,6 +53,28 @@ git switch -c <your-branch-name>
 
 With this branch checked-out, make the desired changes to the package.
 
+### Updating the Vega-Lite version
+
+If your desired changes include updating the Vega-Lite that Altair uses
+you need to change the following section in ``tools/generate_schema_wrapper.py``
+to the desired version number
+
+```python
+SCHEMA_VERSION = {
+    "vega-lite": {"v5": "v5.15.0"},
+}
+```
+
+Then regenerate any automatically generated files
+as described in the next section.
+
+### Updating any automatically generated files
+
+A large part of Altair's code base is automatically generated.
+After you have made your manual changes,
+make sure to run the following to see if there are any changes
+to the automatically generated files: ``tools/generate_schema_wrapper.py``.
```

Comment:
Is this needed to be done manually, or does our workflow do it automatically and add it to the PR?

Reply:
This is already documented in https://github.com/altair-viz/altair/blob/main/NOTES_FOR_MAINTAINERS.md so I think we don't need this section :) But maybe you could link to that markdown file here so it's easier to spot?

Reply:
And yes, it needs to be done manually so that these files can be reviewed as well. In the pipeline/workflow, we then again run the `generate_schema_wrapper.py` to see if it does not produce any additional changes which have not already been committed to make sure that the generated code is always in sync with what is defined in `tools`.

Reply:
Ah yes, there is a maintainers file as well =p I forgot about that one; I will remove this section.

Reply:
I kept a short section to indicate that the schema generation should be run after any changes by pointed to the notes for Vega-Lite updates